Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -317,31 +256,36 @@ protected function emitConfigurationManagerReady(ConfigurationManager $configura
      * If possible just use settings and have them injected.
      *
      * @param string $configurationType The kind of configuration to fetch - must be one of the CONFIGURATION_TYPE_* constants
-     * @param string $configurationPath The path inside the configuration to fetch
-     * @return array|null The configuration or NULL if the configuration doesn't exist
+     * @param string|null $configurationPath The path inside the configuration to fetch
+     * @return mixed|null The configuration or NULL if the configuration doesn't exist
```

Comment:
I've removed the null from `mixed`, thanks.

What you mean by breaking here? I think `getConfiguration()` will return the same as before, that is `null` is returned if the given path is not found or an empty array if the given registered configurationType has no configuration. In the last case, the spaceship operator changes the null to an empty array

Reply:
You are right of course, it could never return null before. The change tricked me but it's really just a correction that makes sense. I thought it would return $configuration as NULL before but that couldn've never happeend because spaceship.